### ISSUE TITLE
[Suggestion] Unlock using PIN or Face ID

### ISSUE THREAD

**Issue by juanbretti** [strongbox-password-safe/Strongbox#220]
Hello,
It would be nice to have another option, for "PIN or Face ID". Being able to unlock with one or the other. So if my face is not available the user needs a different PIN than the one used to unlock the phone. Like the same function available with OneDrive.

Today we have:
- No lock
- PIN Code
- Face ID
- Both
- _PIN or Face ID_ **(addition)**

This option to have it to unlock the **app** and to access the **DB**.
Thank you!

**Comment by mmcguill** [comment]
Yes seems entirely possible

**Comment by juanbretti** [comment]
My issue is: my daughter has my phone PIN, but I don't want her to access my Strongbox. And I would appreciate not to have to type an additional PIN to access Strongbox when I have my face.

Thank you for the consideration.

**Comment by ospring** [comment]
I agree with this request : with face masks , FaceID is not always usable, and unlocking with the pin code in case of faceID failure would be very useful.

**Comment by juanbretti** [comment]
Just a nice **ping** about this feature request :)

**Comment by juanbretti** [comment]
Sorry to push this _issue_ back but it would be great to have this feature.
